src/shared/components/palette/case-link/write-case-link-field.html

```diff
@@ -1,8 +1,8 @@
-<div class="form-group" [ngClass]="{'form-group-error': !caseReferenceControl.valid && caseReferenceControl.dirty}">
+<div class="form-group" [ngClass]="{'form-group-error': !caseReferenceControl.valid && (caseReferenceControl.dirty || caseReferenceControl.touched)}">
```

Comment:
!caseReferenceControl.valid && (caseReferenceControl.dirty || caseReferenceControl.touched)}

this is doing the same job as:

caseReferenceControl.errors && (caseReferenceControl.dirty || caseReferenceControl.touched)